workflows/templates/thamos-advise-template.yaml

```diff
@@ -21,147 +21,39 @@ spec:
-                  else:
-                    _LOGGER.debug(json.loads(exception.body)['error'])
-                    exception_message = json.loads(exception.body)['error']
-
-                  available_exception = True
-          else:
-              available_exception = True
-
-              config_file_exception = True
-
-              message = "No .thoth.yaml was provided."
-              message += " Please add .thoth.yaml to your PR."
-
-              message += "\n\nFor more information have a look at Qeb-Hwt README file:" + \
-              "\n https://github.com/thoth-station/Qeb-Hwt#usage\n"
-
-              message += """
-
-                Example:
-                  host: khemenu.thoth-station.ninja
-                  tls_verify: false
-                  requirements_format: pipenv
-
-                  runtime_environments:
-                  - name: rhel:8
-                    operating_system:
-                      name: rhel
-                      version: "8"
-                    python_version: "3.6"
-                    recommendation_type: stable
-              """
-
-          if available_exception:
-            import hmac
-            import requests
-            import uuid
-
-            payload = {}
-            payload["analysis_id"] = ""
-
-            if config_file_exception:
-              payload["exception"] = message
-            else:
-              payload["exception"] = exception_message
-
-            installation_id = {}
-            installation_id["id"] = "{{inputs.parameters.GITHUB_INSTALLATION_ID}}"
-
-            data = {
-                "action": "finished",
-                "check_run_id": "{{inputs.parameters.GITHUB_CHECK_RUN_ID}}",
-                "installation": installation_id,
-                "base_repo_url": "{{inputs.parameters.GITHUB_BASE_REPO_URL}}",
-                "payload": payload
-            }
-
-            key=os.environ["WEBHOOK_SECRET"]
-            msg = json.dumps(data).encode("UTF-8")
-
-            secret = key.encode("UTF-8")
-            signature = hmac.new(secret, msg, digestmod="sha1")
-
-            headers = {
-                "Accept": "application/vnd.github.antiope-preview+json",
-                "Content-Type": "application/json",
-                "User-Agent": "Workflow/adviser-00000000",
-                "X-GitHub-Delivery": str(uuid.uuid4()),
-                "X-GitHub-Event": "{{inputs.parameters.GITHUB_EVENT_TYPE}}",
-                "X-Hub-Signature": f"sha1={signature.hexdigest()}",
-            }
-
-            print("Headers:\n", headers)
-            print("Data:\n", data)
-
-            WEBHOOK_CALLBACK_URL = os.environ["WEBHOOK_CALLBACK_URL"]
-
-            response = requests.post(WEBHOOK_CALLBACK_URL, data=json.dumps(data), headers=headers)
-            response.raise_for_status()
+      container:
+        name: qeb-hwt-thamos-advise
+        image: workflow-helpers
         env:
+          - name: THOTH_WORKFLOW_TASK
```

Comment:
is there some doc how this works? how do I know which task to call?

Reply:
https://github.com/thoth-station/workflow-helpers/pull/7 doc created @goern this is Qeb-Hwt so it calls `qeb_hwt`